cluster-service/deploy/helm/values.yaml

```diff
@@ -0,0 +1,265 @@
+# TODO: This parameter isn't currently used, but kept to avoid failures in the
+# execution of saasherder. It will be removed once the version of the service
+# that doesn't use it is deployed to all environments.
+debugPort: ""
+
+# Maximum allowed duration for /debug/pprof/ profiling requests, e.g. "30s" or "5m".  "0" means no limit.
+debugMaxDuration: "5m"
+
+# Sets the runtime configuration mode for CS.
+runtimeMode: "aro-hcp"
+
+# Default expiration time assigned to any newly created cluster (e.g. 45m or 72h). 0 means no expiration.
+defaultExpiration: "0"
+
+# Maximum expiration duration possible for any newly created cluster (e.g. 72h means the expiration date cannot be greater than 72h). 0 means no maximum expiration value possible.
+maximumExpiration: "0"
+
+# Duration since cluster creation after which the first notification for stale cluster should be sent.
+firstStaleClusterNotification: "24h"  # 1 day
+
+# Duration since cluster creation after which the second notification for stale cluster should be sent.
+secondStaleClusterNotification: "600h"  # 25 days
+
+# Duration after which a stale cluster can be cleaned up.
+staleClusterAutocleanupWindow: "720h"  # 30 days
+
+# Image Registry
+imageRegistry: ""
+
+# Image Repository
+imageRepository: ""
+
+# Image Tag
+imageTag: ""
+
+# log verbosity level
+logLevel: "debug"
+
+# Number of replicas of the service to run.
+replicas: 1
+
+# Location of the JSON web key set used to verify tokens.
+jwksUrl: "http://localhost"
+
+# Location of the service that issues JSON web tokens.
+tokenUrl: "http://localhost"
+
+# Disable TLS certificate verification on authentication.
+insecure: "false"
+
+# The URL of the gateway.
+gatewayUrl: "http://127.0.0.1:9090"
+
+# Level of access that an app can request to a resource.
+clientScopes: "openid"
+
+# Environment associated with this instance.
+environment: "aro-hcp-dev"
```

Comment:
@geoberle @tony-schndr A note on the environment name.
It shouldn't be longer than 10 chars.
Otherwise, it breaks the cluster's base domain prefix limit. 

For example "aro-hcp-dev" is 11 chars, this means that the cluster's base domain prefix can't be longer than 14 chars while CS allows for it to be <= 15 chars.

Reply:
I will address this in a followup.  We can enforce the length in the config json schema.